src/__tests__/__snapshots__/StateLabel.tsx.snap

```diff
@@ -23,7 +23,7 @@ exports[`StateLabel renders children 1`] = `
   border-style: solid;
   background-color: #28a745;
   color: #ffffff;
-  border-color: rgba(0,0,0,0);
+  border-color: prState.open.border;
```

Comment:
This seems odd.

Reply:
I think this is happening because `prState.open.border` changed from "rgba(0,0,0,0)" to "transparent".

We have have a couple options:
1. Update primer/primitives to use `"rgba(0,0,0,0)"` instead of `"transparent"`
2. Update primer/components to handle `"transparent"` values

Do you have a preference?

Reply:
I don't have a preference - not sure what #2 involves though.

Reply:
None of the changed lines in this PR contain `linear-gradient`, but something to be aware of—`transparent` breaks `linear-gradient` in Safari: https://github.com/github/catalyst/pull/87

Reply:
I'll update primer/primitives to use `rgba(0,0,0,0)` 👍

Reply:
https://github.com/primer/primitives/pull/142